app/javascript/add-another.js

```diff
@@ -85,7 +85,7 @@ const setUpSections = (sectionList) => {
 const setNumbering = (section, counter) => {
   const counterElement = section.querySelector('[data-add-another-role="counter"]')
   if (counterElement) {
-    counterElement.innerHTML = counterElement.dataset.addAnotherNumberFrom === "zero" ? counter : counter + 1;
+    counterElement.innerHTML = counter + 1;
```

Comment:
question - so now the result will still be `: counter 1`, `: counter 2`, `: counter 3` but simplified as there will no longer be `data-add-another-number-from="zero"`